hugegraph-api/src/main/java/com/baidu/hugegraph/auth/StandardAuthenticator.java

```diff
@@ -95,7 +99,7 @@ private String inputPassword() {
             if (!password.isEmpty()) {
                 return password;
             }
-            System.out.println(notEmptyPrompt);
+            LOG.info(notEmptyPrompt);
```

Comment:
seems it was not print in the log file (print on the console is right?) @javeme

Reply:
yes we need System.out sometimes like when using the console